cmake/modules/FindNVJPEG.cmake

```diff
@@ -37,7 +37,8 @@ if(NVJPEG_FOUND)
   endif()
 
   set(CMAKE_REQUIRED_INCLUDES_OLD ${CMAKE_REQUIRED_INCLUDES_OLD})
-  set(CMAKE_REQUIRED_INCLUDES ${CMAKE_CUDA_TOOLKIT_INCLUDE_DIRECTORIES})
+  list(APPEND CMAKE_REQUIRED_INCLUDES "${NVJPEG_INCLUDE_DIR}")
+  list(APPEND CMAKE_REQUIRED_INCLUDES "${CMAKE_CUDA_TOOLKIT_INCLUDE_DIRECTORIES}")
```

Comment:
This is necessary to get `nvjpeg_lossless` to compile with a custom NVJPEG_ROOT_DIR.